lib/matplotlib/widgets.py

```diff
@@ -1839,6 +1839,16 @@ def set_active(self, active):
         if active:
             self.update_background(None)
 
+    def _get_animated_artists(self):
+        """
+        Convenience method to get all animated artists of the figure containing
+        this widget, excluding those already present in self.artists.
+        The returned list is not sorted by 'z_order'.
+        """
+        return tuple([a for ax_ in self.ax.get_figure().get_axes()
+                      for a in ax_.get_children()
+                      if a.get_animated() and a not in self.artists])
```

Comment:
```suggestion
        return tuple(a for ax_ in self.ax.get_figure().get_axes()
                     for a in ax_.get_children()
                     if a.get_animated() and a not in self.artists)
```

if we are going with a tuple might as well not bother with the list in the middle.  I have a slight preference for tuple over list because immutablability is good, but given that his is an internal API either is fine with me!

Reply:
Done!